Essentials/src/main/resources/config.yml

```diff
@@ -484,6 +484,14 @@ auto-afk: 300
 # Set to -1 for no timeout.
 auto-afk-kick: -1
 
+# Define a command the server runs when a player's AFK time elapses.
+# Set to 'none' to use Essentials' default AFK timeout kick behavior.
+#
+# Available placeholders:
+# {USERNAME} - The player's username.
+# {KICKTIME} - The time, in minutes, the player has been AFK for.
```

Comment:
is there a reason we need this variable, wouldn't the time the player has been afk always be the same (the auto-afk-kick above).